engine/client/drivers/docker.go

```diff
@@ -81,7 +81,11 @@ func (d *dockerDriver) create(ctx context.Context, imageRef string, opts *Driver
 		// We only have a tag in the image ref, so resolve it to a digest. The default
 		// auth keychain parses the same docker credentials as used by the buildkit
 		// session attachable.
-		if img, err := remote.Get(ref, remote.WithAuthFromKeychain(authn.DefaultKeychain), remote.WithUserAgent(opts.UserAgent)); err != nil {
+		_, span := otel.Tracer("").Start(ctx, "resolve image")
+		img, err := remote.Get(ref, remote.WithAuthFromKeychain(authn.DefaultKeychain), remote.WithUserAgent(opts.UserAgent))
+		telemetry.End(span, func() error { return rerr })
```

Comment:
nit: should be `return err` here, `rerr` is the return value from the function which isn't set at this point.

Reply:
🙏